### ISSUE TITLE
[Spotify Player] Play My Liked Songs

### ISSUE THREAD

**Issue by nightgolfer** [raycast/extensions#5996]
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

I have an extensive list of Liked Songs, and usually just start that list when I want some background music, as everything on there is something I like.

I can not find any way to play **all** my Liked Songs with the current implementation/version of Spotify Player. "Search Playlist" doesn't find "Liked Songs", probably because Spotify doesn't consider the Liked Songs list to be a playlist. Similarly, I can't find my Liked Songs via "Browse All", "Featured Playlists", and "Just Play..." isn't capable of invoking the list, either.

I would love to be able to just start playing **all** my Spotify Liked Songs from Raycast / Spotify Player, **as if it were a playlist**. The option could be called "Play My Liked Songs".

### Who will benefit from this feature?

Everyone who use Spotify's Liked Songs feature, and like just putting on their Liked Songs for a no-brainer listening session once in a while (or all the time, like me).

### Anything else?

Thanks for the great extension 😃👍

*edit: added some clarification, see words in* **bold**.

**Comment by andreaselia** [comment]
You can use the new Spotify Beta extension found at the link below, it has the ability to play liked songs now 😁 

https://www.raycast.com/mattisssa/spotify-beta

**Comment by dotlouis** [comment]
Hey folks, I love the new beta extension. Well-balanced UX-wise. But I think it does not address @nightgolfer 's issue. Or at least partially only.

There indeed is a new section called "Liked Songs" in the "Your Library" command. But there is no way to play all liked songs. It just plays the one you select.

In Spotify, "Liked Songs" looks like a regular playlist, but it seems not to be a "real" playlist under the hood, as "Liked Songs" does not appear anywhere in the playlists of "Your Library" command.

To sum up, it would be great just to be able to play the "Liked Songs" playlist as a whole and not just one song.

**Comment by nightgolfer** [comment]
Thank you @dotlouis, I couldn't agree more. 

I managed to test the beta, only found what you're describing (which isn't what I was hoping for), and haven't had the time to give feedback. 

What you're describing is exactly what I want — the ability to "Play My Liked Songs", as if it were a playlist (and I agree, it doesn't seem to be treated like an actual playlist, a lot of the normal functions (e.g. 'Playlist Radio') are not there.

**Comment by nightgolfer** [comment]
Still hoping this is possible to implement. Anyone?

**Comment by bkeys818** [comment]
Unforchantly, it's the Spotify API that doesn't allow for this.

See this [community post](https://community.spotify.com/t5/Spotify-for-Developers/Enable-play-back-of-the-quot-Liked-Songs-quot-playlist-through/m-p/4967889).

**Comment by nightgolfer** [comment]
> Unforchantly, it's the Spotify API that doesn't allow for this.
> 
> See this [community post](https://community.spotify.com/t5/Spotify-for-Developers/Enable-play-back-of-the-quot-Liked-Songs-quot-playlist-through/m-p/4967889).

It seems to me that the community post you shared specifically allows for doing what I want — use the Liked Songs context URI (`spotify:user:<username>:collection`) with Start/Resume Playback (https://developer.spotify.com/documentation/web-api/reference/start-a-users-playback).

When I paste `spotify:user:<username>:collection` into the search bar in Spotify, I get sent exactly where I want: my Liked Songs list.
